.github/workflows/remark.yml

```
@@ -34,11 +34,13 @@ jobs:
     - name: Check *.md files
       run: ./node_modules/.bin/remark -u lint -f .
 
-    - name: Linkcheck book
-      run: |
-        rustup toolchain install nightly --component rust-docs
-        curl https://raw.githubusercontent.com/rust-lang/rust/master/src/tools/linkchecker/linkcheck.sh -o linkcheck.sh
-        sh linkcheck.sh clippy --path ./book
+    # FIXME: This is currently disabled because it is not working
+    # see https://github.com/rust-lang/rust-clippy/pull/14339#issuecomment-2695639772
+    # - name: Linkcheck book
+    #   run: |
+    #     rustup toolchain install nightly --component rust-docs
```

```suggestion
        rustup toolchain install --component rust-docs
```
This is just a theory and something you might try.

A recent run of this job that passed installed `nightly-2025-02-27` at the "Running linkchecker..." step: https://github.com/rust-lang/rust-clippy/actions/runs/13620024474/job/38068093874#step:7:45

However, with version 1.28, Rustup no longer installs a toolchain just because it is not installed: https://github.com/rust-lang/rustup/blame/f00c3d1fbcbe8d3ae2411e63ca906bc9b69e43d1/CHANGELOG.md#L9-L11

You have to explicitly install it with a command like `rustup toolchain install`.

---

However, `linkcheck` requires nightly.

https://github.com/rust-lang/rust/blob/e16a049adbf94d610787430b6efdf31d896dc5b6/src/tools/linkchecker/linkcheck.sh#L26

---

OK, I can see what's happening and this time the remark CI succeeded. I'll post additional comment for this PR.